recipes/polars/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "polars" %}
+{% set version = "0.13.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4b349328ae61f6cfd7d29feada03d83baeb8c113ced0902f5e249895f90bfb9b
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - curl https://sh.rustup.rs -sSf | sh -s -- --profile minimal --default-toolchain nightly -y
+    - rustup toolchain install nightly
+    - rustup default nightly
+    # - maturin build -i {{ PYTHON }}
+    # - maturin develop
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}  # [unix]
+    - cmake
+    - make  # [not win]
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - maturin >=0.12
+    - nodejs
+    - numpy >=1.16.0
+    - pandas
+    - pyarrow >=4.0
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.16.0
+    - pytest
+    - pytest-cov
```

Comment:
I think the `pytest` deps should probably be under a `test/requires` section:
* https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-requirements